packages/webapp/next-seo.ts

```diff
@@ -25,3 +27,14 @@ export const defaultOpenGraph: Partial<OpenGraph> = {
     },
   ],
 };
+
+export const getSquadOpenGraph = ({
+  squad,
+}: {
+  squad?: Pick<Source, 'image'>;
+}): Partial<OpenGraph> => ({
+  images:
+    squad?.image && squad.image !== cloudinary.squads.imageFallback
```

Comment:
Since squad image is automatically set to placeholder during creation we actually did not need fallback for squad image.

This check is to make sure if placeholder is set that we show our default SEO image as requested by Product.